### PR TITLE
Nightly build

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .github/workflows/nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          credentials_json: ${{ secrets.GCP_CREDENTIALS_JSON }}
+
+      - name: Setup Testcontainers Cloud Client
+        # Needed for langchain4j-ollama and other modules using testcontainers
+        uses: atomicjar/testcontainers-cloud-setup-action@v1
+        with:
+          token: ${{ secrets.TC_CLOUD_TOKEN }}
+
+      - name: Build with JDK ${{ matrix.java_version }}
+        run: mvn -B -U --fail-at-end ${{ matrix.included_modules }} install
```
Comment (Heezer) [inline .github/workflows/nightly.yaml]: why `install`? is `test` not enough? 🤔

**Comment by Heezer** [comment]
`SqlDatabaseContentRetrieverIT` is failing currently: https://github.com/langchain4j/langchain4j/actions/runs/9233688403/job/25406425318, is known? on the other side is not a blocker of this PR.

**Review by Heezer — DISMISSED** [review]

**Comment by langchain4j** [comment]
> SqlDatabaseContentRetrieverIT is failing currently: https://github.com/langchain4j/langchain4j/actions/runs/9233688403/job/25406425318, is known? on the other side is not a blocker of this PR.

Right, not a blocker, I will check how to make this test more stable

File: .github/workflows/nightly.yaml (hunk 1 above)
Comment (langchain4j) [reply .github/workflows/nightly.yaml]: Good catch! let me fix this

Comment (langchain4j) [reply .github/workflows/nightly.yaml]: lol it should have been `verify` cause `test` does not run integration tests which was the point of this PR

Comment (Heezer) [reply .github/workflows/nightly.yaml]: sorry 😄 but still `install` is too much 😄 good to know, that integration tests are extra categorized

**Pull request merged**